packages/core/src/Site/SiteConfig.js

```diff
@@ -101,6 +101,15 @@ class SiteConfig {
      * @type {Object<string, Object<string, any>>}
      */
     this.deploy = siteConfigJson.deploy || {};
+    /**
+     * @type {boolean}
+     */
+    this.intrasiteLinkValidation = {
+      enabled: (siteConfigJson.intrasiteLinkValidation === undefined
+        || siteConfigJson.intrasiteLinkValidation.enabled === undefined
+        ? true
+        : siteConfigJson.intrasiteLinkValidation.enabled),
+    };
```

Comment:
Hmm, it seems like we need a guard clause to check if `intrasiteLinkValidation` is valid first, before we do `intrasiteLinkValidation.enabled`. Otherwise, we may be reading an undefined property.

Reply:
second line should be
`this.intrasiteLinkValidation.enabled = this.intrasiteLinkValidation.enabled !== false;`*

Reply:
Ah, right. That's a good way to do it. Thanks for the tip!